ctr/const.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+// ctr wide constants
+const (
+	// EXIT_STATUS_OK indicates successful completion
+	EXIT_STATUS_OK = 0
+
+	// EXIT_STATUS_MISSING_ARG indicates failure due to missing argument(s)
+	EXIT_STATUS_MISSING_ARG = 1
```

Comment:
actually, I think @thaJeztah have a point. We usually use camel case for those in Go, good catch!

sorry @HuKeping , do you mind updating it one more time?